packages/data-explorer/src/views/FiltersView.vue

```diff
@@ -90,6 +89,13 @@ export default Vue.extend({
         name: 'main-view',
         query: { ...this.$route.query, filter: this.compressedRouteFilter }
       })
+    },
+    toggleFilterbar () {
+      let filterbarState = this.$route.query.filterbar ? parseInt(this.$route.query.filterbar, 10) : 1
```

Comment:
why the parsing instead of a simple bool

Reply:
The querystring cannot be typed differently than string (afaik). Flipping an int seems more straightforward to me than switching between 'true' and 'false' strings.